resources/maven-archetype-generate.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+
+mvn archetype:generate \
+-DarchetypeGroupId=com.ms3-inc.tavros \
+-DarchetypeArtifactId=camel-openapi-archetype \
+-DarchetypeVersion=0.2.6 \
```

Comment:
thinking maybe we should make this an env var and set it in the pipeline. eventually this might need to be configurable through the args

Reply:
that's a good point, I'll change it.

Reply:
I agree about not having this hardcoded. so not a Jenkins env var but just a groovy var in the pipeline for now? through the params also makes a lot of sense.

Reply:
we resolved this in a call.